src/setup/devnet/index.js

```diff
@@ -590,6 +656,17 @@ export class Devnet {
               ], { stdio: getRemoteStdio() })
             }
 
+            if (this.config.network) {
+              const chain = this.config.network
+              await execa('ssh', [
+                '-o', 'StrictHostKeyChecking=no', '-o', 'UserKnownHostsFile=/dev/null',
+                '-i', '~/cert.pem',
+                      `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}`,
+                      // eslint-disable-next-line
+                      `sed -i "s|\\$BOR_HOME/genesis.json|${chain}|g" node/bor-start.sh`
```

Comment:
Can we use absolute path here for `node/bor-start.sh`?
Also, are you sure those `sed` commands work properly?
Had some issues in the past with Linux machines and had to change `"` to `'` and add additional comma at the end
e.g. https://github.com/maticnetwork/matic-cli/blob/master/.github/workflows/ci.yml#L61

Reply:
Sure. Yep `sed` works fine. I tested it out.